include/tvm/relay/debug_printer.h

```diff
@@ -0,0 +1,49 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file debug_printer.h
+ * \brief A pretty printer for the Relay IR.
+ * As we had not determined a formal syntax yet, right now it is only for debug purpose.
+ */
+#ifndef TVM_RELAY_DEBUG_PRINTER_H_
+#define TVM_RELAY_DEBUG_PRINTER_H_
```

Comment:
since it is debug printer and have nothing to do with the implementation yet, move declarations into ```src/```. 

Add ```DebugPrint(Expr, std::ostream&os)``` to relay/expr.h